app/models/phone_configuration.rb

```diff
@@ -20,4 +20,8 @@ def selection_presenters
     end
     options
   end
+
+  def name
+    :phone if mfa_enabled?
```

Comment:
Can you explain why this is conditional?

Reply:
We only want to count the MFA methods that a user has configured, but I found a better way to do this. Please hold. New commit coming right up.

Reply:
I cleaned it up here: https://github.com/18F/identity-idp/pull/2574/commits/3579a5123fbef3d59910085f4234aefd0c902b88
Let me know what you think